docs/source/reference/math.rst

```diff
@@ -175,6 +175,7 @@ Miscellaneous
    sign
    maximum
    minimum
+   fabs
```

Comment:
Could you move the line to `absolute` and `sign`? (https://github.com/cupy/cupy/blame/v10.0.0/docs/source/reference/math.rst#L4)

Reply:
Okay, sure!